charts/dex/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 name: dex
-version: 0.12.0
-appVersion: "2.35.1"
+version: 0.12.1
+appVersion: "2.35.2"
```

Comment:
Can you please update it to 2.35.3? Thanks!

Reply:
Done :wink: